core/src/main/java/com/netease/arctic/io/reader/CombinedDeleteFilter.java

```diff
@@ -79,7 +80,7 @@ public abstract class CombinedDeleteFilter<T extends StructLike> {
   private final List<DeleteFile> posDeletes;
   private final List<DeleteFile> eqDeletes;
 
-  private Map<String, Set<Long>> positionMap;
+  private Map<String, Roaring64Bitmap> positionMap;
```

Comment:
Maybe we can resue the `org.apache.iceberg.deletesPositionDeleteIndex` here?

Reply:
`Deletes.positionDeleteIndex` reqire datafile path which is not exist in `CombinedDeleteFilter`